src/targets/gpu/prefuse_ops.cpp

```diff
@@ -51,23 +52,26 @@ struct layernorm_base
             auto* pm = mods.front();
             nargs    = pm->get_parameter_names().size();
         }
-        check_shapes{inputs, static_cast<const Derived&>(*this)}.has(nargs + N);
-        auto s = inputs.at(0);
+        nargs += N;
```

Comment:
I think you could just set `nargs` to `N` initially. And then do `nargs  += pm->get_parameter_names().size()`.

Reply:
Modified logic a bit. Thanks.